src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -59,11 +55,25 @@ public Client(String endpoint, Credentials credentials) {
      * @param consentId An identifier to mark the owner of the document handle
      * @return Response from API
      */
-    public JSONObject postDocuments(ContentType contentType, String consentId) throws IOException {
+    public JSONObject createDocument(
+        byte[] content,
+        ContentType contentType,
+        String consentId,
+        Map<String, Object> options
+    ) throws IOException {
         JSONObject jsonBody = new JSONObject();
+        jsonBody.put("content", Base64.getEncoder().encodeToString(content));
         jsonBody.put("contentType", contentType.getMimeType());
         jsonBody.put("consentId", consentId);
 
+        if (options.containsKey("batchId")) {
+            jsonBody.put("batchId", options.get("batchId"));
+        }
+
+        if (options.containsKey("feedback")) {
+            jsonBody.put("feedback", options.get("feedback"));
+        }
```

Comment:
Why not just iterate over options and add all key/value pairs? If the idea is to restrict what is being sent to the API then I suggest making an Options class or something similar and not accept a generic Map<String, Object> as argument.

Reply:
Maybe the best option is to iterate and send all options to the API but document it with docstring as you suggest.